R/database.R

```diff
@@ -1093,8 +1093,10 @@ get_cn_states = function(regions_list,
   cn_matrix = pivot_wider(all_cn, id_cols = "sample_id", names_from = "region_name", values_from = "CN") %>%
     column_to_rownames("sample_id")
 
+  names(cn_matrix) = region_names
```

Comment:
Why was this changed? Is it a bug fix? It seems like it will affect the output.

Reply:
Yes, I had this not working as it was before. I think it might be related to R v3.6 to R v4 shift. Basically it was returning an unnamed vector of numbers (CN states) instead of a named column in a df